templates/test/ci/cluster-template-prow-dual-stack.yaml

```diff
@@ -0,0 +1,4637 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    cni: ${CLUSTER_NAME}-calico
+  name: ${CLUSTER_NAME}
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 10.244.0.0/16
```

Comment:
If we're using these for e2e tests, can we create env variables to substitute the CIDRs like in the custom-vnet and private cluster templates?

Reply:
Do you have an example? I don't see environment variables for CIDS in the custom-vnet or private cluster prow templates.

Reply:
Do we have a scenario where we would need to overwrite the CIDR block for the tests?